src/App/Hovedside/Sak/SisteSaker/SisteSaker.tsx

```diff
@@ -1,75 +1,37 @@
-import { Link, To, useLocation } from 'react-router-dom';
+import React, { useContext, useEffect } from 'react';
+import { useLocation } from 'react-router-dom';
 import { OrganisasjonsDetaljerContext } from '../../../OrganisasjonDetaljerProvider';
 import './SisteSaker.css';
 import { useSaker } from '../useSaker';
-import { loggNavigasjon } from '../../../../utils/funksjonerForAmplitudeLogging';
 import amplitude from '../../../../utils/amplitude';
-import { BodyShort, Heading } from '@navikt/ds-react';
+import { Heading, Tag } from '@navikt/ds-react';
 import { useSessionStateForside } from '../Saksoversikt/useOversiktSessionStorage';
 import { SakSortering } from '../../../../api/graphql-types';
-import { PaperplaneIcon } from '@navikt/aksel-icons';
 import { OrganisasjonerOgTilgangerContext } from '../../../OrganisasjonerOgTilgangerProvider';
 import { sorted } from '../../../../utils/util';
 import { Set } from 'immutable';
+import { InternalLenkepanelMedLogging } from '../../../../GeneriskeElementer/LenkepanelMedLogging';
 
 const ANTALL_FORSIDESAKER: number = 3;
 
-interface SakerLenkeProps {
-    ikon: ReactNode;
-    overskrift: string;
-    undertekst: string;
-    to: To;
-}
-
-const SakerLenke = ({ ikon, overskrift, undertekst, to }: SakerLenkeProps) => {
-    const [hover, setHover] = useState(false);
-    const ikonId = 'ikon-id' + overskrift.toLowerCase().replace(' ', '-');
-    const lenkeId = 'lenke-id' + overskrift.toLowerCase().replace(' ', '-');
-
-    addEventListener('mouseover', (e) => {
-        const target = e.target as HTMLElement;
-        if (target.id === ikonId) {
-            setHover(true);
-        } else if (target.id === lenkeId) {
-            setHover(true);
-        } else {
-            setHover(false);
-        }
-    });
-    return (
-        <div className="saker-lenke">
-            <Link
-                tabIndex={-1}
-                id={lenkeId}
-                className={'saker-lenke__ikon' + (hover ? ' saker-lenke__ikon__hover' : '')}
-                to={to}
-                onClick={() => {
-                    scroll(0, 0);
-                    loggNavigasjon('saksoversikt', 'se alle saker', location.pathname);
-                }}
-            >
-                {ikon}
-            </Link>
-
-            <Link
-                className={
-                    'saker-lenke_headerlenke' + (hover ? ' saker-lenke_headerlenke__hover' : '')
-                }
-                to={to}
-                onClick={() => {
-                    scroll(0, 0);
-                    loggNavigasjon('saksoversikt', 'se alle saker', location.pathname);
-                }}
-            >
-                <Heading id={ikonId} size={'small'}>
-                    {overskrift}
-                </Heading>
-            </Link>
-            <BodyShort className="saker-lenke__undertekst ">{undertekst}</BodyShort>
-        </div>
-    );
-};
+export const Saksikon = () => (
```

Comment:
Hvorfor ikke importere ikon som før?

Reply:
Importert ikon hadde css-styling for å lage ikon fra Figma, mens nytt ikon er komplett svg.